packages/daemon/src/model/plugin.ts

```diff
@@ -9,7 +9,17 @@ providerWrapper([
   }
 ]);
 
-export class PluginModel extends Model {
+export interface IPluginModel {
```

Comment:
Why do you make this a new interface which is same as the `PluginModel` class?

Reply:
Is there any way to defined the output with model?